src/Microsoft.ML.AutoML/Utils/StringEditDistance.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Microsoft.ML.AutoML.Utils
+{
+    public static class StringEditDistance
+    {
+        public static int GetEditDistance(string first, string second)
```

Comment:
Can be more specific with the naming:
```suggestion
        public static int GetLevenshteinDistance(string first, string second)
```

Also allows for future alternative measures added later.

Reply:
Done